core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -277,10 +263,12 @@ fun DropdownTextField(
     modifier: Modifier = Modifier,
     title: String,
     items: List<String>,
-    selected: (String) -> Unit
+    selected: (String) -> Unit,
```

Comment:
I think you just need one value or selected and one onSelectedOption.

Reply:
update